platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/resolveengine/DependencyGraphResolver.java

```diff
@@ -154,9 +154,20 @@ private ModuleConflictResolver<ComponentState> createModuleConflictResolver(Conf
     }
 
     private static List<CapabilitiesConflictHandler.Resolver> createCapabilityConflictResolvers(CapabilitiesResolutionInternal capabilitiesResolutionRules) {
+
+        // The order of these resolvers is significant. They run in the declared order.
```

Comment:
This has nothing to do with attribute matching and is related to the internal workings of how capability conflict handling is performed.

Reply:
I understand.  I just strongly think we need a single, step-by-step, written description of "here's what happens when you resolve".  I'd change the title and expand the scope of the documentation to include this information, rather than excluding the information because it doesn't match the current title/scope.

Reply:
IMO these two informations have not the same public:
* A plugin author needs to understand variant matching to create custom variants, integrate them in an ecosystem, and, in general, make it work
* The info here is really for developers needing to work on the dependency resolution engine of Gradle

To be clear: I am not against that second point having its own external documentation. But we would need to make sure it does not become stale.

Reply:
I won't be updating the documentation in this PR